### ISSUE TITLE
I can not install BMR Package

### ISSUE THREAD

**Issue by khmahmood** [kthohr/BMR#26]
I have been trying to install  the"BMR" package, but unfortunately, I failed. I received this statement:  

Skipping install of 'BMR' from a github remote, the SHA1 (33c0fbb6) has not changed since last install.
  Use `force = TRUE` to force installation.
Please, any help will be really appreciated.

**Comment by kthohr** [comment]
That means the package is already installed on your computer... But like the error message indicated, try
```
install_github("kthohr/BMR",force=TRUE)
```
Works fine on my system.

**Comment by khmahmood** [comment]
Thank you so much for the quick reply. It is supposed that the package has been installed after I applied the command that you have mentioned. But, when I was trying to apply an example from the package just to check, I received " could not find function "BVARS" and another function.  I have tried again but this time I received "
Error in install_github("kthohr/BMR", force = TRUE) : 
  could not find function "install_github"
In addition: Warning message:
S3 methods ‘plot.Rcpp_bvarm’, ‘plot.Rcpp_bvars’, ‘plot.Rcpp_bvarcnw’, ‘plot.Rcpp_bvarinw’, ‘plot.Rcpp_bvartvp’, ‘plot.Rcpp_dsge_gensys’, ‘plot.Rcpp_dsgevar_gensys’, ‘forecast.Rcpp_bvarm’, ‘forecast.Rcpp_bvars’, ‘forecast.Rcpp_bvarcnw’, ‘forecast.Rcpp_bvarinw’, ‘forecast.Rcpp_cvar’, ‘mode_check.Rcpp_dsge_gensys’, ‘mode_check.Rcpp_dsgevar_gensys’ were declared in NAMESPACE but not found .
Error in if (file.exists(dest) && file.mtime(dest) > file.mtime(lib) &&  : 
  missing value where TRUE/FALSE needed.
I have followed the instructions that are written in http://www.kthohr.com/bmr.html .
What should i do now? 
Again thank you so much.

**Comment by kthohr** [comment]
`install_github` is a devtools function.

 Run
```
install.packages("devtools")
library(devtools)
install_github("kthohr/BMR",force=TRUE)
```
Then try one of the the examples from the README; for example,
[https://github.com/kthohr/BMR#bvar-example](https://github.com/kthohr/BMR#bvar-example)

**Comment by khmahmood** [comment]
Thanks. I have tried the commands that you have just sent them, but when I was trying to run install.packages("devtools"), 
I have received:
Error in install.packages : missing value where TRUE/FALSE needed

**Comment by khmahmood** [comment]
Now I tried again, I have been asked to select the installation of Rtools, is this normal?
I have previously installed it.

**Comment by kthohr** [comment]
That means R(Studio) cannot find your Rtools installation. Follow these instructions carefully:

[https://cran.r-project.org/doc/manuals/R-admin.html#The-Windows-toolset](https://cran.r-project.org/doc/manuals/R-admin.html#The-Windows-toolset)

Install R and Rtools to `C:\` (not Program Files), and ensure your system PATH is set correctly. This has nothing to do with my package so I cannot assist you further.

**Comment by khmahmood** [comment]
Thank you so much.

**Comment by khmahmood** [comment]
Hi and sorry to bother you again, it seems that everything has worked as I have successfully applied the example.  For example, this is the mean
> bvar_obj$alpha_pt_mean
             [,1]
 [1,]  1.83585970
 [2,]  0.62762200
 [3,]  0.32703091
 [4,] -0.32759365
 [5,]  0.23849524
 [6,]  0.84891052
 [7,]  0.33339724
 [8,]  0.59962042
 [9,] -0.04291800
[10,]  0.08400967
But when I am trying to apply it using my own data, I have received the following:

could not find function "BVARM"

**Comment by kthohr** [comment]
What, exactly, do you mean by 'when I am trying to apply it using my own data'? Paste the exact command you're invoking before you receive that error. 

Note that there is `BVARM` function in the package; that refers to a previous iteration of the package. The examples given in the README, and in the tests folder, illustrate how to properly use the package.

**Comment by khmahmood** [comment]
Thank you so much for the quick response. Forgive me as I do not have much knowledge abut coding and computer science. I normally use any package in R directly, but it seems this one is different. Thanks for the information, I will read the tests folder. In your example, about Minssota prior, you have chosen (0.5) for the coefficient of the variable of interest in each equation and the others had the value zero. I have seen some researchers set this value to one instead of 0.5 and the other coefficients are set to zero, does this mean we are free to chose the value for the prior for the main variable in each equation?
Thanks a lot.

**Comment by kthohr** [comment]
You're free to choose any values, really. A value of one implies a random walk prior, which might make sense If, say, your data are in levels (e.g., log-GDP). If you think your data are stationary but somewhat persistent (perhaps you have first-differenced the series), then a prior between zero and one makes sense. You can then adjust the hyperparameters to reflect how confidant you are in these prior values.

**Comment by khmahmood** [comment]
Thank you so much.

**Comment by sanusinwu** [comment]
Good day sir, I am trying to install BMR package in R. I have carefully followed all the procedures. Please sir, but got this error message. Error: Failed to install 'BMR' from GitHub:
  (converted from warning) installation of package ‘C:/Users/DRKASA~1/AppData/Local/Temp/RtmpGshzE3/file43d029f95112/BMR_0.11.0.tar.gz’ had non-zero exit status

**Comment by Sijan-t** [comment]
I am facing the same problem as @sanusinwu . I have been trying to install the package on computer(mac OS 10.14) for a week now. I tried installing with: install_github("kthohr/BMR"); it says non-zero exit status (already installed xcode etc.). 
Then I tried using install.packages("BMR", repo = 'https://mac.R-project.org'), it still doesn't work. It says "package ‘BMR’ is not available (for R version 3.6.1)".
Please let us know what we can do. 
Thank you so much!

**Comment by masoumehsolgi** [comment]
hi,
im trying to use the "BMR " package
unfortunatlly after i installed dvtools (install.packages("devtools")) and run this command devtools::install_github("kthohr/BMR",force=TRUE)
i have this error
Warning in install.packages :
  package ‘BMR’ is not available (for R version 4.0.2)
what shoild i do ,please help me.

**Comment by Abhayilu** [comment]
install.packages("devtools")
library(devtools)
install_github("kthohr/BMR")
After running these above code when I run library command, i get the following error 
Error in library("BMR") : there is no package called ‘BMR’
library("BMR")
